apps/explorer/src/theme/ThemeProvider.tsx

```diff
@@ -22,6 +33,8 @@ export const ThemeProvider = ({ children }: PropsWithChildren) => {
       // Compute the app colour pallette using the passed in themeMode
       ...themePalette,
       ...fontPalette,
+      // Add widget mode flags
+      ...widgetMode,
```

Comment:
There is no widget mode in explorer, why do we add the value?

Reply:
Addressed